bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/niri/Niri.java

```diff
@@ -797,20 +793,6 @@ public ROIDescription(ROIDescription roid) {
         }
 
         /**
-         * Return the x start pixel.
-         */
-        public int getXStart() {
-            return _xStart;
-        }
-
-        /**
-         * Return the y start pixel.
-         */
-        public int getYStart() {
-            return _yStart;
-        }
-
-        /**
```

Comment:
:+1: if the full build compiles and tests pass, call it good. There is very little reflection left in OCS, thankfully.